### PR TITLE
Package coq-hierarchy-builder.1.7.0

### PR DESCRIPTION
**Pull request by coqelpibot**
### `coq-hierarchy-builder.1.7.0`
High level commands to declare and evolve a hierarchy based on packed classes
Hierarchy Builder is a high level language to build hierarchies of algebraic structures and make these
hierarchies evolve without breaking user code. The key concepts are the ones of factory, builder
and abbreviation that let the hierarchy developer describe an actual interface for their library.
Behind that interface the developer can provide appropriate code to ensure retro compatibility.

---
* Homepage: https://github.com/math-comp/hierarchy-builder
* Source repo: git+https://github.com/math-comp/hierarchy-builder
* Bug tracker: https://github.com/math-comp/hierarchy-builder/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

### CODE REVIEW

File: released/packages/coq-hierarchy-builder/coq-hierarchy-builder.1.7.0/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+and abbreviation that let the hierarchy developer describe an actual interface for their library.
+Behind that interface the developer can provide appropriate code to ensure retro compatibility."""
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: ["Cyril Cohen" "Kazuhiko Sakaguchi" "Enrico Tassi"]
+license: "MIT"
+tags: "logpath:HB"
+homepage: "https://github.com/math-comp/hierarchy-builder"
+bug-reports: "https://github.com/math-comp/hierarchy-builder/issues"
+depends: [
+  "coq-elpi" {(>= "2.0") | = "dev"}
```
Comment (palmskog) [inline released/packages/coq-hierarchy-builder/coq-hierarchy-builder.1.7.0/opam:16]:
```suggestion
  "coq-elpi" {>= "2.0"}
```
@gares the `dev` part is redundant without an upper bound before it.

File: released/packages/coq-hierarchy-builder/coq-hierarchy-builder.1.7.0/opam [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+]
+conflicts: ["coq-hierarchy-builder-shim"]
+build: [
+  [make "build"]
```
Comment (palmskog) [inline released/packages/coq-hierarchy-builder/coq-hierarchy-builder.1.7.0/opam:20]: Is there a reason to not use make-level parallelism here?

Comment (CohenCyril) [reply released/packages/coq-hierarchy-builder/coq-hierarchy-builder.1.7.0/opam:20]: I don't think there is

Comment (palmskog) [reply released/packages/coq-hierarchy-builder/coq-hierarchy-builder.1.7.0/opam:20]: OK, let's try and see what CI says.

File: released/packages/coq-hierarchy-builder/coq-hierarchy-builder.1.7.0/opam [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+license: "MIT"
+tags: "logpath:HB"
+homepage: "https://github.com/math-comp/hierarchy-builder"
+bug-reports: "https://github.com/math-comp/hierarchy-builder/issues"
+depends: [
+  "coq-elpi" {>= "2.0"}
+]
+conflicts: ["coq-hierarchy-builder-shim"]
+build: [
+  [make "build"]
```
Comment (palmskog) [inline released/packages/coq-hierarchy-builder/coq-hierarchy-builder.1.7.0/opam:20]:
```suggestion
  [make  "-j%{jobs}%" "build"]
```

**Pull request merged**